templates/documents/upload_files_modal.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'documents/shared/modal_body.html' %}
+
+{% block modal_body %}
+    <div class="mt-2 flex flex-col gap-5">
+        <input
+            x-ref="fileUpload"
+            name="files"
+            @change="handleFileChange($event)"
+            type="file"
+            multiple
+            class="file-input file-input-bordered w-full max-w-xs"
+            accept=".txt,.pdf,.jpg,.png"/>
```

Comment:
why only these?

Reply:
Oh this was just to get started with something. I [added more](https://github.com/dimagi/open-chat-studio/pull/1259/commits/fb74a2fedf02be12af4b904c7fc63ad4e4f445b2)